<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>789</width>
    <height>321</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: 
#FBF8F1
;</string>
  </property>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>20</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{ 
	background-color: #92A9BD; color:#FFFFFF;
font: 17pt &quot;Mulish&quot;;color:#fff; 
	border-radius: 13.25760269165039px;
}

 QPushButton:hover {
	background-color: #7C99AC; color:#FFFFFF;
font: 17pt &quot;Mulish&quot;;color:#fff; 
	border-radius: 13.25760269165039px;
 }</string>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QLabel" name="hasil">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>250</y>
     <width>321</width>
     <height>41</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;Mulish&quot;;color:#000; </string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>20</y>
     <width>341</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20pt &quot;Mulish&quot;;color:#1D2D5E; </string>
   </property>
   <property name="text">
    <string>Insert Signature</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="nilaip">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>130</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 15pt &quot;Mulish&quot;;color:#1D2D5E; 
height: 47.76695251464844px;
width: 376.5159912109375px;
left: 538px;
top: 229px;
border-radius: 13.25760269165039px;
border:2px solid #7C99AC;
padding-left:15px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Input P</string>
   </property>
  </widget>
  <widget class="QPushButton" name="insertSig">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>190</y>
     <width>321</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{ 
	background-color: #92A9BD; color:#FFFFFF;
font: 17pt &quot;Mulish&quot;;color:#fff; 
	border-radius: 13.25760269165039px;
}

 QPushButton:hover {
	background-color: #7C99AC; color:#FFFFFF;
font: 17pt &quot;Mulish&quot;;color:#fff; 
	border-radius: 13.25760269165039px;
 }</string>
   </property>
   <property name="text">
    <string>Insert Signature</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nilaiq">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>130</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 15pt &quot;Mulish&quot;;color:#1D2D5E; 
height: 47.76695251464844px;
width: 376.5159912109375px;
left: 538px;
top: 229px;
border-radius: 13.25760269165039px;
border:2px solid #7C99AC;
padding-left:15px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Input Q</string>
   </property>
  </widget>
  <widget class="QPushButton" name="uploadButton">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>80</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{ 
	background-color: #92A9BD; color:#FFFFFF;
font: 17pt &quot;Mulish&quot;;color:#fff; 
	border-radius: 13.25760269165039px;
}

 QPushButton:hover {
	background-color: #7C99AC; color:#FFFFFF;
font: 17pt &quot;Mulish&quot;;color:#fff; 
	border-radius: 13.25760269165039px;
 }</string>
   </property>
   <property name="text">
    <string>Upload</string>
   </property>
  </widget>
  <widget class="QLabel" name="fileName">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>80</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;Mulish&quot;;color:#000; </string>
   </property>
   <property name="text">
    <string>File Belum Diupload!</string>
   </property>
  </widget>
  <widget class="QPushButton" name="genButton">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>130</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{ 
	background-color: #92A9BD; color:#FFFFFF;
font: 17pt &quot;Mulish&quot;;color:#fff; 
	border-radius: 13.25760269165039px;
}

 QPushButton:hover {
	background-color: #7C99AC; color:#FFFFFF;
font: 17pt &quot;Mulish&quot;;color:#fff; 
	border-radius: 13.25760269165039px;
 }</string>
   </property>
   <property name="text">
    <string>Generate Key</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
